<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mwClothEditor</class>
 <widget class="QMainWindow" name="mwClothEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cloth Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFrame" name="qfSceneNodes">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <layout class="QVBoxLayout" name="vlSceneNodes">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabClothEditor">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>350</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tiAttr">
         <attribute name="title">
          <string>Attrs</string>
         </attribute>
         <layout class="QGridLayout" name="glPreset">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="vlAttr">
            <property name="spacing">
             <number>0</number>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tiVtxMap">
         <attribute name="title">
          <string>VtxMap</string>
         </attribute>
         <layout class="QGridLayout" name="glVtxMap">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="vlVtxMap">
            <property name="spacing">
             <number>0</number>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="mLockParam">
     <property name="title">
      <string>Lock Param</string>
     </property>
     <addaction name="miLockAll"/>
     <addaction name="miUnlockAll"/>
    </widget>
    <widget class="QMenu" name="mFilters">
     <property name="tearOffEnabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Filters</string>
     </property>
    </widget>
    <addaction name="miToolTips"/>
    <addaction name="miRefreshUi"/>
    <addaction name="mFilters"/>
    <addaction name="mLockParam"/>
   </widget>
   <widget class="QMenu" name="mFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QMenu" name="mOpenRootPath">
     <property name="title">
      <string>Open Root Path</string>
     </property>
     <addaction name="miXplorer"/>
     <addaction name="miXterm"/>
    </widget>
    <addaction name="miSetRootPath"/>
    <addaction name="mOpenRootPath"/>
   </widget>
   <addaction name="mFiles"/>
   <addaction name="mOptions"/>
  </widget>
  <action name="miNCloth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>nCloth</string>
   </property>
  </action>
  <action name="miNRigid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>nRigid</string>
   </property>
  </action>
  <action name="miRfSceneNodes">
   <property name="text">
    <string>Scene Nodes</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="miToolTips">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ToolTips</string>
   </property>
  </action>
  <action name="miSetRootPath">
   <property name="text">
    <string>Set Root Path</string>
   </property>
  </action>
  <action name="miXplorer">
   <property name="text">
    <string>Xplorer</string>
   </property>
  </action>
  <action name="miXterm">
   <property name="text">
    <string>Xterm</string>
   </property>
  </action>
  <action name="miLockAll">
   <property name="text">
    <string>Lock All</string>
   </property>
  </action>
  <action name="miUnlockAll">
   <property name="text">
    <string>Unlock All</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="miRefreshUi">
   <property name="text">
    <string>Refresh Ui</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
